rules/docker_config.sh.tpl

```diff
@@ -23,7 +23,7 @@ main() {
   # Expand a tar file with the repo if needed
   %{EXPAND_REPO_CMD}
   # Call the script produced by the docker_build rule to load the image
-  %{LOAD_IMAGE_SH}
+  ./%{LOAD_IMAGE_SH}
```

Comment:
Do we want to include this change in this PR?

Reply:
I'll undo this.